scripts/regression.py

```diff
@@ -745,24 +746,13 @@ def _update_one(params):
 
       shutil.copy(surelog_log_filepath, golden_log_filepath)
 
-      # lines = []
-      # with open(surelog_log_filepath, 'rt', encoding='cp850') as istrm:
-      #   for line in istrm:
-      #     line = line.rstrip('\n')
-      #     line = line.replace('/regression/', '/tests/')
-      #     lines.append(line)
-      #   istrm.close()
-      #
-      # while not lines[-1]:
-      #   lines = lines[:-1]
-      # lines.append('')
-      #
-      # with open(golden_log_filepath, 'wt', encoding='cp850') as ostrm:
-      #   for line in lines:
-      #     ostrm.write(line)
-      #     ostrm.write('\n')
-      #   ostrm.flush()
-      #   ostrm.close()
+      # On Windows, fixup the line endings
+      if platform.system() == 'Windows':
+        with open(golden_log_filepath, 'rt', encoding='cp850') as istrm:
```

Comment:
Wouldn't UTF-8 be more robust in case there is any unicode in the output ? cp850 sounds very 1990ies...

Reply:
That was immediate solution when I saw an encoding issue but that it didn't work. I remember the issue to being only on MSYS2.

That said, the logs have changed quite a bit and I remember you making a specific change to remove non-ascii characters from log.

I will try changing it to utf-8 across the script and see if anything falls. Will follow up with a new PR once the current open PRs are merged and dust settles.